deliverables/README.md

```diff
@@ -31,38 +38,116 @@ What is the role of Javascript in a web page?
 
 #### meta
 Why do we separate HTML and CSS?  
+Abstracting CSS from the HTML is a best practice that makes the HTML  
+easier to read, and therefore easier to debug/change.
 
 #### HTML
 What are classes and IDs (and how are they different)?  
+Classes are CSS selectors for identifying multiple elements whereas IDs identify a single element.
+
 What are elements?  
+Elements are basic components that a web page is made of.
+
 What are tags?  
+Tags define the beginning and end of an element.
+
 What are attributes?  
+Attributes modify/specify behavior of elements.
+
 What are forms?  
+Forms define user input areas to be sent to the webserver.
```

Comment:
... and also the URL that the data should be sent to, in the `action` attribute of the `form` tag.